index.html

```diff
@@ -4,7 +4,293 @@
+                  d="M58.2837 35.8363C59.38 33.98 60 32 60 29.6063C60 24.1044 55.3477 18.9088 49.2725 18.9088H44.7125C45.3275 17.3075 45.8187 15.3925 45.8187 13.0913C45.8187 3.992 41.1075 0 33.91 0C26.2091 0 26.6484 11.8666 24.94 13.575C22.0966 16.4184 18.7381 21.8809 16.345 24H4C1.79087 24 0 25.7909 0 28V58C0 60.2091 1.79087 62 4 62H12C13.8616 62 15.426 60.7282 15.8722 59.0062C21.4359 59.1314 25.2547 63.9987 38.0975 63.9987C39 63.9987 40 64 40.875 64C50.5146 64 54.8732 59.0721 54.9925 52.0838C56.6574 49.7806 57.5299 46.6935 57.16 43.71C58.3918 41.4035 58.868 38.6671 58.2837 35.8363ZM50.565 42.565C52.135 45.2062 50.7225 48.7412 48.8225 49.7612C49.785 55.8587 46.6215 57.9987 42.1825 57.9987H37.455C28.5001 57.9987 22.7014 53.2713 16 53.2713V30H17.365C20.91 30 25.8625 21.1387 29.1825 17.8175C32.7275 14.2725 31.5462 8.36375 33.91 6C39.8187 6 39.8187 10.1225 39.8187 13.0913C39.8187 17.9875 36.2738 20.1813 36.2738 24.9088H49.2725C51.9112 24.9088 53.9888 27.2725 54 29.6362C54.0112 31.9987 52.3975 34.3625 51.2163 34.3625C52.9024 36.1819 53.2626 40.017 50.565 42.565ZM11 54C11 55.6569 9.65687 57 8 57C6.34312 57 5 55.6569 5 54C5 52.3431 6.34312 51 8 51C9.65687 51 11 52.3431 11 54Z"
+                  fill="blue"
+                />
+              </svg>
+              <p class="section3__text-after-svg">Book & Enjoy</p>
+              </div>
+            </div>
+        </div>
+          </div>
+    
+      </section>
+      <section class="section4">
+        <div class="section4_box_for_place">
+          <div class="section4__place section4_place">
+            <p class="section4__name-city section4_name-city">Spain</p>
+            <h3 class="section4__title section4_title">Grand Spain Madrid</h3>
+            <p class="section4__text-title section4_text-title">
+              But I must explain to you how all this mistaken idea of account to
+              denouncing pleasure and praising pain was account born and I
+              account will give you a complete account of the system
+            </p>
+            <div class="section4_button">
+              <button class="section4__button" type="submit">book now</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="section5">
+        <div class="section5_box_for_place">
+          <div class="section5__place section5_place">
+            <p class="section5__name-city section5_name-city">Spain</p>
+            <h3 class="section5__title section5_title">Grand Spain Madrid</h3>
+            <p class="section5__text-title section5_text-title">
+              But I must explain to you how all this mistaken idea of account to
+              denouncing pleasure and praising pain was account born and I
+              account will give you a complete account of the system
+            </p>
+            <div class="section5_button">
+              <button class="section5__button" type="submit">book now</button>
+            </div>
+          </div>
```

Comment:
такий блок теж вже є, тому краще винести стилі в окремий клас, напр. так
```suggestion
          <div class="section5__place card">
            <p class="card__subtitle">Spain</p>
            <h3 class="card__title">Grand Spain Madrid</h3>
            <p class="card__text">
              But I must explain to you how all this mistaken idea of account to
              denouncing pleasure and praising pain was account born and I
              account will give you a complete account of the system
            </p>
            <button class="button card__button" type="submit">book now</button>
          </div>
```